application/models/TodoModel.php

```diff
@@ -3,4 +3,10 @@ class TodoModel extends CI_Model{
         public function insertList($data){
             return $this->db->insert("todo",$data);
         }
+        public function fetchTask(){
+            $this->db->select("*");
+            $this->db->from("todo");
```

Comment:
you can change this to `$this->db->get('todo');`. That way, you don't have to include 'from'.